packages/flame/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## [next]
+## [1.0.0-rc9]
```

Comment:
I normally keep the next empty to avoid further conflicts. (not worth changing imo just a thought for the future)

Reply:
I will add a commit after I generate the tag, so the next don't show on pub

Reply:
ahh didn't know it would show up there. I won't do that anymore then haha